slack_sdk/web/base_client.py

```diff
@@ -59,6 +59,9 @@ def __init__(
     ):
         self.token = None if token is None else token.strip()
         """A string specifying an `xoxp-*` or `xoxb-*` token."""
+        # self.base_url = base_url
```

Comment:
same as above

Reply:
Thank you for your feedback, I just pushed the updates.